test/lambda/fixtures/aws_api_http_test_data.json

```diff
@@ -14,7 +14,7 @@
       "x-forwarded-proto": "https"
     },
     "requestContext": {
-      "accountId": "XXXXXXXXXXXX",
+      "accountId": "000000000000",
```

Comment:
REVIEW NOTE: I heavily use the "XXX" string in in-progress/draft work to mark things that are *incomplete* and should not be merged to master. It is helpful for my development to not have any "XXX" false positives in the repo, if it can be avoided.